src/sentry/static/sentry/app/icons/iconClose.tsx

```diff
@@ -3,29 +3,41 @@ import React from 'react';
 import {IconProps} from 'app/types/iconProps';
 import theme from 'app/utils/theme';
 
-export const IconClose: React.FC<IconProps> = ({
-  color: providedColor = 'currentColor',
-  size: providedSize = 'sm',
-  circle: providedCircle = false,
-  ...props
-}: IconProps) => {
-  const color = providedColor;
-  const size = theme.iconSizes[providedSize] ?? providedSize;
+export const IconClose = React.forwardRef(
+  (
+    {
+      color: providedColor = 'currentColor',
+      size: providedSize = 'sm',
+      circle: providedCircle = false,
+      ...props
+    }: IconProps,
+    ref: React.Ref<SVGSVGElement>
+  ) => {
```

Comment:
Looks like we're losing the component name in snapshots, I think we can get it back if we use a named function like below -- not sure how necessary this is though

```suggestion
export const IconClose = React.forwardRef(
  function IconClose(
    {
      color: providedColor = 'currentColor',
      size: providedSize = 'sm',
      circle: providedCircle = false,
      ...props
    }: IconProps,
    ref: React.Ref<SVGSVGElement>
  ) {
```

Reply:
Sure I'll add that